google-cloud-core/src/test/java/com/google/cloud/BaseWriteChannelTest.java

```diff
@@ -103,8 +101,12 @@ public void testClose() throws IOException {
   @Test
   public void testValidateOpen() throws IOException {
     channel.close();
-    thrown.expect(ClosedChannelException.class);
-    channel.write(ByteBuffer.allocate(42));
+    try {
+      channel.write(ByteBuffer.allocate(42));
+      Assert.fail();
+    } catch (ClosedChannelException ex) {
+      assertNotNull(ex);
+    }
   }
```

Comment:
thrown exception is never null.
I think the test will look much better if you rewrite it as

```
@Test(expected = ClosedChannelException.class)
public void testValidateOpen() throws IOException {
    channel.close();
    channel.write(ByteBuffer.allocate(42));
}
```

Reply:
done